hadoop-ozone/ozonefs/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSInputStream.java

```diff
@@ -134,4 +135,20 @@ public void testO3FSMultiByteRead() throws IOException {
     Assert.assertTrue(Arrays.equals(value, data));
     inputStream.close();
   }
+
+  @Test
+  public void testO3FSByteBufferRead() throws IOException {
+    FSDataInputStream inputStream = fs.open(filePath);
```

Comment:
i would really love to see this wrapped in a try with block to ensure resource is closed properly. But I see that all test cases in the file follows the same pattern so maybe we can have a separate jira to correct that.

Reply:
Thanks @jojochuang for the suggest.
JIRA created for this issue. ([HDDS-2723](https://issues.apache.org/jira/browse/HDDS-2723))